packages/medusa/src/commands/migrate.ts

```diff
@@ -11,7 +12,9 @@ const main = async function ({ directory }) {
   args.shift()
   args.shift()
 
+  const action = args[0]
   const container = await initializeContainer(directory)
+  container.register("logger", asValue(Logger))
```

Comment:
todo: the initializeContainer already register the logger, so you should be fine here

Reply:
Makes sense!